File: ArtworkPostServices/src/main/java/com/artauction/artworkpostservices/Dtos/ArtworkPost/ArtworkPostDto.java
package com.artauction.artworkpostservices.Dtos.ArtworkPost;

import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Getter
@Setter
public class ArtworkPostDto {
    private Integer id;
    private String title;
    private String description;
    private BigDecimal initialPrice;
    private BigDecimal buyNowPrice;
    private LocalDateTime startDate;
    private LocalDateTime endDate;
    private String image;
    private String artistName;
    private String categoryName;
    private String[] tags;
}

File: Project/Backend/src/ArtAuction.Application/DTOs/ArtworkPost/ArtworkPostDto.cs
using ArtAuction.Application.DTOs.Tag;
using ArtAuction.Application.Entities;
using Microsoft.AspNetCore.Http;

namespace ArtAuction.Application.DTOs.ArtworkPost;

public class ArtworkPostDto
{
    // Attributes
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; } 
    public decimal InitialPrice { get; set; }
    public decimal BuyNewPrice { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }
    public string Image { get; set; }
    public string ArtistName { get; set; }
    public string CategoryName { get; set; }
    public string[] Tags { get; set; }
}